pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -113,6 +190,181 @@ def load(fnames, tag=None, inst_id=None, file_cadence=dt.timedelta(days=1),
         return data, meta
 
 
+def load_xarray(fnames, tag=None, inst_id=None,
+                labels={'units': ('units', str), 'name': ('long_name', str),
+                        'notes': ('notes', str), 'desc': ('desc', str),
+                        'plot': ('plot_label', str), 'axis': ('axis', str),
+                        'scale': ('scale', str),
+                        'min_val': ('value_min', float),
+                        'max_val': ('value_max', float),
+                        'fill_val': ('fill', float)},
+                epoch_name='Epoch', meta_processor=None,
+                meta_translation={}, drop_meta_labels=None):
+    """Load NASA CDAWeb CDF files into an xarray Dataset.
+
+    Parameters
+    ----------
+    fnames : pandas.Series
+        Series of filenames
+    tag : str or NoneType
+        tag or None (default=None)
+    inst_id : str or NoneType
+        satellite id or None (default=None)
+    labels : dict
+        Dict where keys are the label attribute names and the values are tuples
+        that have the label values and value types in that order.
+        (default={'units': ('units', str), 'name': ('long_name', str),
+        'notes': ('notes', str), 'desc': ('desc', str),
+        'min_val': ('value_min', np.float64),
+        'max_val': ('value_max', np.float64),
+        'fill_val': ('fill', np.float64)})
+    epoch_name : str or NoneType
+        Data key for epoch variable.  The epoch variable is expected to be an
+        array of integer or float values denoting time elapsed from an origin
+        specified by `epoch_origin` with units specified by `epoch_unit`. This
+        epoch variable will be converted to a `DatetimeIndex` for consistency
+        across pysat instruments.  (default='time')
+    meta_processor : function or NoneType
+        If not None, a dict containing all of the loaded metadata will be
+        passed to `meta_processor` which should return a filtered version
+        of the input dict. The returned dict is loaded into a pysat.Meta
+        instance and returned as `meta`. (default=None)
+    meta_translation : dict or NoneType
+        Translation table used to map metadata labels in the file to
+        those used by the returned `meta`. Keys are labels from file
+        and values are labels in `meta`. Redundant file labels may be
+        mapped to a single pysat label. If None, will use
+        `default_from_netcdf_translation_table`. This feature
+        is maintained for compatibility. To disable all translation,
+        input an empty dict. (default={})
```

Comment:
```suggestion
        input an empty dict. (default=None)
```

Because empty dicts to bad things as default kwargs.

Reply:
I agree that an empty dict is not a good default kwarg. However, I don't think that the default netcdf translation table is appropriate either

Reply:
You misunderstand, I think.  An empty dict as a default won't work.  There are pointer issues, so you won't get an empty dict.  To get an empty dict as input, you need to set the default to 'None' and then have an if statement that assigns the empty dict if the input value is None.  This is what I'm recommending you do.